src/routes/Navbar.svelte

```diff
@@ -0,0 +1,64 @@
+<script>
+	import NavLink from './NavLink.svelte';
+
+	let navLinksVisible = false;
+</script>
+
+<nav class="nav">
+	<button class="nav-button" on:click={() => (navLinksVisible = !navLinksVisible)}>☰</button>
```

Comment:
I think it might be better to use `&#9776;` instead of the unicode directly, or possibly an SVG? Not a blocking matter though, can be addressed later.